### PR TITLE
Have a new release of azure cni/npm at 1.2.6 cache on VHD

### PR DESCRIPTION
**Pull request by paulgmiller**
Just published in this build def
https://dev.azure.com/AzureContainerUpstream/Kubernetes/_build/results?buildId=18234&view=results

### CODE REVIEW

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -147,10 +147,10 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/kubernetes/v1.20.2/windowszip/v1.20.2-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://acs-mirror.azureedge.net/azure-cni/v1.1.8/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.1.8.zip",
             "https://acs-mirror.azureedge.net/azure-cni/v1.2.0/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.0.zip",
             "https://acs-mirror.azureedge.net/azure-cni/v1.2.0_hotfix/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.0_hotfix.zip",
             "https://acs-mirror.azureedge.net/azure-cni/v1.2.2/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.2.zip"
+            "https://acs-mirror.azureedge.net/azure-cni/v1.2.6/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.6.zip"
```
Comment (mattstam) [inline vhdbuilder/packer/configure-windows-vhd.ps1:153]: Missing comma here

**Review by mattstam — CHANGES_REQUESTED** [review]
comma problem other than that looks good!

File: vhdbuilder/packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (paulgmiller) [reply vhdbuilder/packer/configure-windows-vhd.ps1:153]: excellent

**Review by mattstam — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -105,13 +105,15 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
...
 VNET_CNI_VERSIONS="
+1.2.6
 1.2.0_hotfix
 1.2.0
-1.1.8
 "
 for VNET_CNI_VERSION in $VNET_CNI_VERSIONS; do
     VNET_CNI_PLUGINS_URL="https://acs-mirror.azureedge.net/azure-cni/v${VNET_CNI_VERSION}/binaries/azure-vnet-cni-linux-amd64-v${VNET_CNI_VERSION}.tgz"
     downloadAzureCNI
+    VNET_CNI_PLUGINS_URL="https://acs-mirror.azureedge.net/azure-cni/v${VNET_CNI_VERSION}/binaries/azure-vnet-cni-swift-linux-amd64-v${VNET_CNI_VERSION}.tgz"
```
Comment (paulgmiller) [inline vhdbuilder/packer/install-dependencies.sh:115]: this won't work as we don't have swift < 1.2/4

**Comment by paulgmiller** [comment]
Moving over to different PR that is based out of this rep so vhd gates can pass.

**Pull request closed without merging**